### PR TITLE
Console Calculator - All challenges completed

### PR DESCRIPTION
**Pull request by JerryCJY**
Class library and OOP concepts have me using a lot of time but finally, I completed them all!

Let me know what I can improve on!

### CODE REVIEW

File: JerryCheong.Calculator/Program.cs [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+                numInput2 = Console.ReadLine();
+
+                double cleanNum2 = 0;
+                while (string.IsNullOrEmpty(numInput2))
+                {
+                    Console.Write("Input cannot be empty. Please enter a numeric value or choose an operator:");
+                    numInput2 = Console.ReadLine();
+                }
+                while (!double.TryParse(numInput2, out cleanNum2) && !Regex.IsMatch(numInput2, "[r|t|sine|cosine|tangent]"))
```
Comment (javedkhan2k2) [inline JerryCheong.Calculator/Program.cs:58]: if numInput2 = 's' is entered the program didn't act accordingly.
There is also a bug when you catch an exception in your CalculatorLibrary.
I will suggest that you should entered different invalid inputs for the second input.
Please fix this issue.

File: JerryCheong.Calculator/Program.cs [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+                Console.WriteLine("\tr - SquareRoot");
+                Console.WriteLine("\tt - TenPower");
+                Console.WriteLine("\tsine - Sin");
+                Console.WriteLine("\tcosine - Cos");
+                Console.WriteLine("\ttangent - Tan");
+
+                numInput2 = Console.ReadLine();
+
+                double cleanNum2 = 0;
+                while (string.IsNullOrEmpty(numInput2))
```
Comment (javedkhan2k2) [inline JerryCheong.Calculator/Program.cs:53]: If an exception occurred then the numInput2 contains the previous value.

**Review by javedkhan2k2 — COMMENTED** [review]
Please fix the issues in the comments test the program for valid and invalid input and submit it again for review.
In File Program.cs line # 58 and #53.
Please read the comments for details.
If you need help send me a message.

File: JerryCheong.Calculator/Program.cs (hunk 1 above)
Comment (JerryCJY) [reply JerryCheong.Calculator/Program.cs]: I have fixed the issue, please have a look again. Thank you!

**Comment by javedkhan2k2** [comment]
@JerryCJY well done. Good structured code with challenges also implement. Good luck with your next Project.

**Pull request closed without merging**